core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -98,15 +100,15 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
   @Test
   def testCreate(): Unit = {
     createAndWaitTopic(new TopicCommandOptions(
-      Array("--partitions", "2", "--replication-factor", "1", "--topic", testTopicName)))
+      brokerOptions ++ Array("--partitions", "2", "--replication-factor", "1", "--topic", testTopicName)))
```

Comment:
We didn't `checkArgs` before, so it's ok to not pass the `--bootstrap-server` option. Since we make it as required argument now, we need to pass the option explicitly.